internal/request/valid_outbound.go

```diff
@@ -18,16 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
```

Comment:
did you mean `validator_outbound.go`? `valid_outbound.go` seems a bit confusing - like, are the others not valid? :P